2.cellprofiler_processing/CellProfiler_input/CFReT_project_CL.cppipe

```diff
@@ -228,12 +228,31 @@ MeasureTexture:[module_num:15|svn_version:'Unknown'|variable_revision_number:7|s
     Measure whole images or objects?:Objects
     Texture scale to measure:3
 
-ExportToDatabase:[module_num:16|svn_version:'Unknown'|variable_revision_number:28|show_window:False|notes:[]|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+MeasureImageQuality:[module_num:16|svn_version:'Unknown'|variable_revision_number:6|show_window:True|notes:[]|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
```

Comment:
There are very small differences between the two files.

Firstly, the `.ccproj` file is the default file type when creating a project in CellProfiler. I include it in this module for the user to use for the GUI since that is the type of file it expected as input. The parameters within the GUI are different like the output path and you have to drag in the images.

The `.cppipe` file is something that you can export from the GUI. That is the expected format when using the command line and the output path has to be set differently. I do think that in the future, I will be trying to use CellProfiler headless so I will always have the `.cppipe` file for users to go over.